addons/im_livechat/static/src/embed/common/thread_model_patch.js

```diff
@@ -138,6 +138,15 @@ patch(Thread.prototype, {
         return super.showUnreadBanner;
     },
 
+    get composerHide() {
+        const step = this.chatbot?.currentStep;
+        return (
+            this.livechat_active === false ||
+            this.chatbot?.completed ||
```

Comment:
Not sure but checking whether the operator was found or in the case of chatbot is completed might resolve the issue where the composer gets disabled after the chatbot is completed and human operator is added.